### PR TITLE
feat(imagecardformcontent): support clearing the image

### PR DESCRIPTION
**Pull request by scottdickerson**
Closes #

**Summary**

- feat(imagecardformcontent): support a button that clears the selected image
- refactor(dashboardeditor): better support selecting the image from the gallery

**Acceptance Test (how to verify the PR)**

- Verify the story selecting an image from the gallery works
- Verify the new story of CardEditor for Image sends the onChange event correctly

### CODE REVIEW

File: src/components/CardEditor/CardEditForm/_card-edit-form.scss [hunk 1]
```diff
@@ -157,13 +157,31 @@
...
   &-btn {
     width: 100%;
   }
+  &-clear-button {
+    position: absolute;
+    right: $spacing-05;
+    margin-bottom: 0;
+    padding: 0 $spacing-02;
+    bottom: $spacing-04;
+    color: $ui-05;
```
Comment (tay1orjones) [inline src/components/CardEditor/CardEditForm/_card-edit-form.scss:174]: I think this positioning might be redundant if we used the `size="field"` prop on the button. I believe it's intended to size buttons for use in fields like this. Nothing to hold this up for, but would be nice to in the future have it "full bleed" without any padding like other buttons. pagination for example

image

**Review by tay1orjones — APPROVED** [review]
It's an edge case we might not need to address right now, but If the filename is longer than the input can display, should we truncate it and provide a tooltip on hover?

The close/remove button is transparent currently so I think the text will just run behind it

**Review by JoelArmendariz — APPROVED** [review]
LGTM

**Comment by scottdickerson** [comment]
thanks @tay1orjones I made that change

**Pull request merged**